package.json

```diff
@@ -21,7 +21,8 @@
     "link-check": "repo-link-check -c config/link-check/config.yml",
     "link-check-diff": "repo-link-check -c config/link-check/config.yml -d master",
     "link-check-dev-server": "repo-link-check -c config/link-check/config.yml -r http://localhost:3000",
-    "link-check-exclude": "repo-link-check -c config/link-check/config.yml --unused-patterns-only"
+    "link-check-exclude": "repo-link-check -c config/link-check/config.yml --unused-patterns-only",
+    "prepare": "husky install"
```

Comment:
Husky works different after v8! [See husky docs](https://typicode.github.io/husky/#/?id=husky-4-to-8-cli)